resources/views/posts/create-post.blade.php

```diff
@@ -0,0 +1,45 @@
+@php
+    /** @var $group String: The id of the group */
+@endphp
+
+<x-app-layout>
+    <x-slot name="header">
+        <h2 class="font-semibold text-xl text-gray-800 leading-tight">
+            {{ __('Create Post') }}
+        </h2>
+    </x-slot>
+    
+    @if ($group)
+    <form action="/create-post?group={{$group}}" method="post">
+        @csrf
+        <x-slot name="header">
+            <h2 class="font-semibold text-xl text-gray-800 leading-tight">
+                {{ $group }} - New Post
+            </h2>
+        </x-slot>
+        <x-input-label for="title">
+            Post Title
+        </x-input-label>
+        <x-text-input 
+            id="title" 
+            name="title"
+            type="text"
+            class="block mt-1 w-full"
+            required />
+        <x-input-label for="content">
+            Post Content
+        </x-input-label>
+        <x-textarea 
+            id="content" 
+            name="content"
+            rows="4"
+            maxlength="500"
+            class="block mt-1 w-full" 
+            placeholder="Insert the post's content..."
+            required/>
```

Comment:
This will make a hidden `input` that will store our group id as we need it :)
```suggestion
            required/>
      <input type="hidden" name="group_id" value="{{ $group->id}}" />
```

Reply:
Solved,